### DISCUSSION TITLE
TUG site - non-conservativeForces files

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#155] [General]
Hello 

are the files that are listed under "NonconservativeForces" directories for the different satellites (on the TUG website) a sum of the non-conservative forces (accelerations) that are being used in the variational equation scripts for reduced dynamic orbits? if yes how are the files then generated?

Thank you very much for the clarification and help

Cheers
Jean

**Comment by tmayerguerr** [comment]
Hi Jean,

The non-conservative forces are computed with [SimulateAccelerometer](https://groops-devs.github.io/groops/html/SimulateAccelerometer.html) and used the same forces as in [PreprocessingVariationalEquation](https://groops-devs.github.io/groops/html/PreprocessingVariationalEquation.html)->forces->[miscAccelerations](https://groops-devs.github.io/groops/html/miscAccelerationsType.html).

regards
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten
txs for the feedback and clarification.  That goes along with what we thought they were.

Cheers
Jean
